scripts/repackage_bundle.sh

```diff
@@ -41,8 +41,13 @@ do
         rm -rf "./node_modules/ibm_db/installer/clidriver"
     fi
 
-    # Remove native code from Keytar module bundled with the SCS plugin
```

Comment:
isn't this still true?

Reply:
Yes, but I thought it was confusing to say "remove native code" when we also add native code here now 😋 The original behavior is still happening and should be included in "clean up unwanted binaries"